docs/application/native/guides/internationalization/i18n.md

```diff
@@ -869,9 +878,67 @@ using a Number format for the PluralFormat locale.
 
 ### Define Custom Plural Rules
 
-To add custom plural rules to the PluralFormat, create a Plural Rules object and add it in PluralFormat constructor.
+To add custom plural rules to the PluralFormat, create a PluralRules object and add it in PluralFormat constructor.
 If you also pass a locale object to constructor, this locale will affect formatted number in the returned message texts.
 
+<a name="date_interval"></a>
+## Usages of Date Intervals
+
+### Create Date Interval Object
+
+The DateInterval API (in [mobile](../../api/mobile/latest/group__CAPI__BASE__UTILS__I18N__DATE__INTERVAL__MODULE.html) and [wearable](../../api/wearable/latest/group__CAPI__BASE__UTILS__I18N__DATE__INTERVAL__MODULE.html) applications) allows you to [create date intervals](#date_interval_examples).
+
+DateInterval module represents time period between pair of `i18n_udate` values. For example, this type contains
+number of milliseconds from 1 January 1970 00:00 UTC to desired date.
+
+### Format Date Intervals
+
+The DateIntervalFormat API (in [mobile](../../api/mobile/latest/group__CAPI__BASE__UTILS__I18N__DATE__INTERVAL__FORMAT__MODULE.html) and [wearable](../../api/wearable/latest/group__CAPI__BASE__UTILS__I18N__DATE__INTERVAL__FORMAT__MODULE.html) applications) allows you to [format created date intervals](#date_interval_examples).
+
+Purpose of this module is to enable formatting date intervals in a language-independent manner. DateIntervalFormat module formats a date interval into text as compactly as possible.
+For example, the date interval format from "Jan 11, 2008" to "Jan 18, 2008" is
+"Jan 11-18, 2008" for English.
+
+There is no structural information in date time patterns. For any punctuation or string literals in a date time pattern,
+you do not know whether it is just a separator, a prefix, or a suffix. Without such information,
+it is difficult to generate a sub-pattern (or super-pattern) using algorithm. Therefore, formatting a date interval is pattern-driven.
+
+A skeleton:
```

Comment:
Suggestion:
To maintain parallelism, can we rephrase to:
- Keeps only the field pattern letter and ignores all other parts in a pattern, such as  space, punctuation marks, and string literals.
- Hides the order of fields.
- Hides a field's pattern letter length.

Please verify for technical accuracy.

Reply:
For now I left it as it was, because last information about letter length is not accurate. Specifying skeleton there is a possibility to hide a pattern length, but it is not obligatory for developer to use it.

Reply:
OK.